### PR TITLE
Contactme

### PR DESCRIPTION
**Pull request by bentleybiscuit**
I still need to fix the responsiveness of the navbar so it's all visible on mobile viewpoint.

In this review, please look at the about me and contact me sections as I have been working on them together (doh!)

Thanks.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,9 +14,9 @@
 			<div>
 				<ul>
 
-					<li>about me</li>
```
Comment (LukeChaston) [inline index.html:16]: These lines go off the image and merge with the gradient at the side when I shrink the image. Perhaps fix this by making it stay in one place?

File: index.html [hunk 2]
```diff
@@ -45,14 +46,13 @@ <h3>On fire, no bugs</h3>
 			</div>
 			
 			<section class="container">
-				<h1>About me:</h1>
+				<h1 id="aboutMe">About me:</h1>
 				<p>This is where I'm going to tell you all about myself and the skills that I am developing on the Full Stack Track at Mayden Academy.  There's the minor detail of the online test, and then I'll be a certified scrum master, which I'll tell you about too.  I'm based in Bath and am looking for my first role as a junior developer: I'm available from the middle of December 2019.</p>
```
Comment (LukeChaston) [inline index.html:50]: This font size shrinks when I go to the mobile option, making it quite difficult to read.

File: index.html [hunk 3]
```diff
@@ -2,8 +2,9 @@
 <html lang="en-GB">
 	
 	<head>
-			<link rel="stylesheet" type="text/css" href="normalize.css">
```
Comment (LukeChaston) [inline index.html:5]: Why the repeat?

File: index.html [hunk 4]
```diff
@@ -13,9 +14,9 @@
 			<div>
 				<ul>
```
Comment (LukeChaston) [inline index.html:15]: This section is tad bloated.

**Review by LukeChaston — CHANGES_REQUESTED** [review]
Just a few changes, mostly relating to the mobile phone stuff. But fairly good apart from that. Well done. You can do it!

File: index.html (hunk 1 above)
Comment (bentleybiscuit) [reply index.html:16]: Yes, I noted this in my comment on the pull request: I'll resolve it next.

File: index.html (hunk 2 above)
Comment (bentleybiscuit) [reply index.html:50]: Ok that's useful, I wasn't sure if i had made it too small.  I'll skip the smallest media query on the font size.

File: index.html (hunk 1 above)
Comment (bentleybiscuit) [reply index.html:16]: Oh, I had misunderstood what you meant.  Yes, the text would be better fixed in an ideal world. I'll see if I have time!

**Comment by LukeChaston** [comment]
All looks good

**Pull request merged**